docs/docs/tutorial/chapter2/routing-params.md

```diff
@@ -796,10 +796,10 @@ export const Failure = ({ error }: CellFailureProps) => (
   <div style={{ color: 'red' }}>Error: {error.message}</div>
 )
 
-export const Success = ({ article }: CellSuccessProps<ArticleQuery>) => (
+export const Success = ({ article }: CellSuccessProps<ArticleQuery>) => {
   // highlight-next-line
-  <Article article={article} />
-)
+  return <Article article={article} />
```

Comment:
@Tobbe What about lines 764 & 766?

Reply:
Bah. JS. Who even cares? 😜
Just kidding. Of course those should be updated too.

Reply:
https://github.com/redwoodjs/redwood/pull/6235

Reply:
https://github.com/redwoodjs/redwood/pull/6235